Frontend/app/mentii.config.ts

```diff
@@ -4,12 +4,23 @@ export class MentiiConfig {
 
   getRootUrl():string {
     var url:string = 'http://api.mentii.me';
```

Comment:
Idk if that is worth a code change lol.. other side of the coin is it will default to prod if we add another instance and make a mistake and we don't want that...